docs/diagrams/ci/AutomatedTestingActivityDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
```

Comment:
Are diagrams necessary for non-application based extraneous utilites? Either way, I believe these diagrams need to be saved as images inside docs/images/unicash

Reply:
Yes, I would imagine it's useful to help other developers understand the flow of our GH Actions given that some steps only occur on specific runners.

Noted on the shift, I'll still create a separate folder under `unicash` regardless to avoid clutter

Reply:
That works, good idea, thinking about it maybe later on we can create subfolders for each related group of commands so the folder doesn't get cluttered